neurom/core/tests/test_soma.py

```diff
@@ -79,15 +79,6 @@
     [11, 22, 33, 44, 2, 1, 1]
 ]
 
-BIFURCATING_SOMA_PTS = [
```

Comment:
Yes, the h5 soma is a contour but that means a _single_ section. Indeed the existence of bifurcations oppose to the h5 definition in https://bbpcode.epfl.ch/browse/code/common/morphology-documentation/
```
Soma: there is only one soma section
```
so there should be a test to ensure that bifurcations are not allowed (it doesn't have to be in this commit surely but at least there should be an issue for this).